openlibrary/macros/UserEditRow.html

```diff
@@ -9,16 +9,20 @@
     </div>
 $if unique:
     <div class="input">
-        <input type="text" id="$name" name="$name" value="$value"/>
+        <span id="inputs">
```

Comment:
I like the approach you've taken here to generalize this.  Could we improve this solution by dropping the `inputs` ID here and using a class instead?